moveit_core/collision_detection_fcl/test/test_fcl_collision_detection.cpp

```diff
@@ -318,39 +314,39 @@ TEST_F(FclCollisionDetectionTester, AttachedBodyTester)
   robot_state.update();
 
   res = collision_detection::CollisionResult();
-  crobot_->checkSelfCollision(req, res, robot_state, *acm_);
+  c_env_->checkSelfCollision(req, res, robot_state, *acm_);
   ASSERT_FALSE(res.collision);
 
   pos1.translation().x() = 5.01;
   shapes::Shape* coll = new shapes::Box(.1, .1, .1);
-  cworld_->getWorld()->addToObject("coll", shapes::ShapeConstPtr(coll), pos1);
+  c_env_->getWorld()->addToObject("coll", shapes::ShapeConstPtr(coll), pos1);
   res = collision_detection::CollisionResult();
-  cworld_->checkRobotCollision(req, res, *crobot_, robot_state, *acm_);
+  c_env_->checkRobotCollision(req, res, robot_state, *acm_);
   ASSERT_TRUE(res.collision);
 
   acm_->setEntry("coll", "r_gripper_palm_link", true);
   res = collision_detection::CollisionResult();
-  cworld_->checkRobotCollision(req, res, *crobot_, robot_state, *acm_);
+  c_env_->checkRobotCollision(req, res, robot_state, *acm_);
   ASSERT_TRUE(res.collision);
 }
 
-TEST_F(FclCollisionDetectionTester, DiffSceneTester)
+TEST_F(FclCollisionDetectionTester, DISABELD_DiffSceneTester)
```

Comment:
What's the reason for this being disabled?

Reply:
Also typo

Reply:
Good catch Bryce! I disabled this when I was developing the `unified_env` and had not yet implemented / combined all of the constructors. I enabled the test and it passes.